server/liveblog/syndication/blogs.py

```diff
@@ -76,20 +78,40 @@ def _create_blogs_syndicate(blog_id, consumer_blog_id):
     consumer_id = str(consumer['_id'])
     if out_service.is_syndicated(consumer_id, blog_id, consumer_blog_id):
         return api_error('Syndication already sent for blog "{}".'.format(blog_id), 409)
-    "{}"
-    syndication_id = out_service.post([{
+
+    if not start_date:
+        # TODO: Find a way to force value to None, as it's ignoring schema settings.
+        # we are forced to set a date in the past, as python-eve is saving by default to datetime.datetime.now().
+        start_date = datetime.datetime(2010, 1, 1, 0, 0, 0)
+
+    doc = {
         'blog_id': blog_id,
         'consumer_id': consumer_id,
-        'consumer_blog_id': consumer_blog_id
-    }])[0]
+        'consumer_blog_id': consumer_blog_id,
+        'start_date': start_date
+    }
+
+    syndication_id = out_service.post([doc])[0]
     syndication = out_service.find_one(_id=syndication_id, req=None)
```

Comment:
unfortunately it's required as post() method is returning the document id, not the whole document, and we need the whole document to get and put the generated token in the response.